src/main.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include "../include/screen.h"
+#include "../include/flags.h"
+#include "../include/game.h"
+
+int main(int argc, char* argv[]){
+
+  // Parse the CLI flags
+  Flags flags(argc, argv);
+  flags.Parse();
+
+  // Create a new screen and handle the screen by the main Game class
+  Screen screen(800,600,"PongPlusPlus-Client");
+  screen.Init();
+  Game game(&screen);
```

Comment:
Passing stack variables around with pointers like this is generally bad practice. `screen` should be on the heap (using `Screen screen = new Screen(800,600,"PongPlusPlus-Client");`) and be wrapped by a `std::unique_ptr`.